composer.json

```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0"
+        "php": "^8.0",
+        "laravel-zero/framework": "^9.0",
+        "guzzlehttp/guzzle": "^7.4",
+        "illuminate/http": "^9.0",
```

Comment:
Sorry for the late answer, I've completely missed it. Thanks for the PR, but I'm just wondering if we need all those dependencies in a production build or if they should be enough as dev dependency?